app/main.py

```diff
@@ -1,3 +1,79 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[int, float, "Distance"]) -> "Distance":
```

Comment:
```suggestion
    def __add__(self, other: int, float, Distance) -> Distance:
```

please fix everywhere 
if you get an error, try
```python
from __future__ import annotations
```